src/test/java/seedu/address/logic/commands/BmiCommandTest.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.FitNus;
+import seedu.address.model.ReadOnlyFitNus;
+
+public class BmiCommandTest {
```

Comment:
Are you sure this is how Bmi works in fitNUS? I suggest relooking at our codebase and familiarising yourself with how we create and calculate BMI. Feel free to reply the group if you're still facing issues understand our code.